### ISSUE TITLE
crafting apron: result is XL/XS, the recipe is not

### ISSUE THREAD

**Issue by Brambor** [CleverRaven/Cataclysm-DDA#78574]
### Describe the bug

bad:
![image](https://github.com/user-attachments/assets/57835863-57bd-4ec4-ae6b-91c7ecbad956)

less bad (badness filed separately #78573)
![image](https://github.com/user-attachments/assets/51244135-3a4d-4b30-9b75-817838b2b30f)

### Attach save file

N/A

### Steps to reproduce

1. Set all skills to 10.
2. Open crafting GUI and search apron.
3. Select the broken recipes.

### Expected behavior

XL/XS in recipe name corresponds to result name (also to the crafting result).

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.5247 (22H2)
- Game Version: 0.G-15607-gf384fab805-dirty [64-bit] master was a7c5b6f04b
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by moxian** [comment]
earlier report:
* #72193

**Comment by Brambor** [comment]
True, I feel like this is more concise so I will leave it open. But if anyone else feels like closing this, do it.
